app/konnect/dev-portal/audit-logging/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Audit Logging in Dev Portal
+badge: enterprise
+content_type: concept
+---
+
+{% include_cached /md/konnect/audit-logging/audit-log-overview.md %}
+
+{:.note}
+> **Note:** Dev Portal audit logs are set up and managed separately from org-wide {{site.konnect_short_name}} audit logs. For more information about how to configure audit logging for {{site.konnect_short_name}}, see [Set up an audit log webhook for Dev Portal](/konnect/dev-portal/audit-logging/webhook/).
```

Comment:
What do you think about not having this be a note? I think it might fit nice before the include, like an intro to the include.